src/panels/Panels.tsx

```diff
@@ -46,6 +26,8 @@ export default function Panels() {
         return <CharmsPanel />;
       case "charm-actions":
         return <CharmActionsPanel />;
+      case "config":
+        return <ConfigPanel />;
```

Comment:
This panel was handled in `EntityDetails` but is now handled here with all the others.